game_eggs/steamcmd_servers/squad/egg-squad-wine.json

```diff
@@ -0,0 +1,244 @@
+        },
+        {
+            "name": "Query Port",
+            "description": "",
+            "env_variable": "QUERY_PORT",
+            "default_value": "27165",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Advertise Server",
+            "description": "Have the server report to the public server list ( true \/ false )",
+            "env_variable": "ADVERTISE",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Reserved Slots",
+            "description": "The number of reserved slots for admins \/ mods",
+            "env_variable": "RESERVEDSLOTS",
+            "default_value": "0",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Prevent Team Change If Unbalanced",
+            "description": "This will prevent players from changing teams if they're already unbalanced ( true \/ false )",
+            "env_variable": "PTCIU",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Enforce Team Balance",
+            "description": "This will FORCE team balance if the teams are too uneven ( true \/ false )",
+            "env_variable": "TEAMBAL",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Record Demos",
+            "description": "This will record demos of the players ( true \/ false )",
+            "env_variable": "RECORDDEMOS",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "STEAM USER",
+            "description": "Specify Steam User or leave it Blank for anonymous",
+            "env_variable": "STEAM_USER",
+            "default_value": "anonymous",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "max:30",
+            "field_type": "text"
+        },
+        {
+            "name": "STEAM PASSWORD",
+            "description": "Specify Steam Password or leave it Blank for anonymous",
+            "env_variable": "STEAM_PASS",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "max:30",
```

Comment:
should be string validated and steam passwords can be longer then 30 and also specify it can be nullable.